pkg/controllers/override/util.go

```diff
@@ -112,12 +113,37 @@ func lookForMatchedPolicies(
 	return policies, false, nil
 }
 
+type helpData struct {
+	gvk       schema.GroupVersionKind
+	sourceObj *unstructured.Unstructured
+}
+
+func getHelpDataFromFedObj(fedObj fedcorev1a1.GenericFederatedObject) (*helpData, error) {
+	gvk, err := getGVKFromFederatedObject(fedObj)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get gvk from fedObj: %w", err)
+	}
+	sourceObj, err := fedObj.GetSpec().GetTemplateAsUnstructured()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get sourceObj from fedObj: %w", err)
+	}
+	return &helpData{
+		gvk:       gvk,
+		sourceObj: sourceObj,
+	}, nil
+}
+
+// 集群名称，和对应的所有patches
```

Comment:
please translate it

Reply:
sorry, this content is a comment for my own viewing only and has been deleted.